src/Ocelot/Request/Mapper/RequestMapper.cs

```diff
@@ -41,29 +42,35 @@ public async Task<Response<HttpRequestMessage>> Map(HttpRequest request, Downstr
             }
         }
 
-        private static async Task<HttpContent> MapContent(HttpRequest request)
+        private static bool BodyIsSupported(string method)
         {
-            if (request.Body == null || (request.Body.CanSeek && request.Body.Length <= 0))
+            return method != HttpMethod.Get.Method

```

Comment:
I don't agree, you can put a body in a get request, it's not standard & recommended, but it's possible.
Ocelot must remain open to this kind of "exotic" case.

Reply:
Ray,
Will be a good idea to provide config option (settings) to ignore (to include) bodies of mentioned web methods to follow (not follow) web standards?

Reply:
Maybe this should be configured from the configuration so that we could change it easily if needed

Reply:
One more configuration option for a false problem.
Ocelot is designed to meet a wide range of needs (for example, we're using it to deal with an old API that didn't respect any standards or rules), so it's already complex enough to add a new option.
That's just my opinion.

Reply:
https://www.rfc-editor.org/rfc/rfc7231#section-4.3.1

As for the GET with body, we could mention elasticsearch...

Reply:
@RaynaldM and @ggnaegi 
As I see, we have a compromise to allow body for GET requests. In this case the PT doesn't have much sense...
Hmmm... Let me think a bit more...